python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
OK. I think OSX10.10 comes with python 2.7.6. Maybe there is a way to use the system python.